DirectProgramming/DPC++FPGA/Tutorials/Features/read_only_cache/src/read_only_cache.cpp

```diff
@@ -0,0 +1,164 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
+#include <chrono>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;
+namespace oa = sycl::ext::oneapi;
```

Comment:
The oa seems a little too terse for a sample.     maybe ext_oneapi?   That shortens it by 7 characters and still highlights more explicitly it's a oneAPI extension feature.

Reply:
Addressed